src/app/docs/page/page-docs.component.html

```diff
@@ -0,0 +1,31 @@
+<sky-docs-demo-page
+  moduleName="SkyPageModule"
+  moduleSourceCodePath="src/app/public/modules/page/"
+  pageTitle="Page"
+>
+  <sky-docs-demo-page-summary>
+    The page component wraps the contents of an index.html file....
```

Comment:
How about something like...

> The page component resets the SPA's background to white, and provides an easy way to override styling.

Depending on what we come up for design guidelines might have an impact on this sentence.

Reply:
Looks good.